Sources/TensorFlow/Optimizer.swift

```diff
@@ -355,3 +355,100 @@ public class AdaGrad<Model: Layer>: Optimizer
         update(&model.allDifferentiableVariables, along: direction)
     }
 }
+
+/// ADADELTA optimizer.
+///
+/// ADADELTA is a more robust extension of AdaGrad that adapts learning rates
```

Comment:
This block of API documentation has lines whose length varies significantly from lines to lines. Could you make sure  each line is filled with words until it reaches 100 columns? This is what I meant by suggesting "fit to size". Hope this clarifies things!